<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dehumEffDialog</class>
 <widget class="QDialog" name="dehumEffDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Effectiveness/NTU</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="effButton">
        <property name="text">
         <string>Effectiveness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="effLine"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="NTUButton">
          <property name="text">
           <string>NTU Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="estNTUButton">
          <property name="text">
           <string>Estimate
NTU Value</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="NTULine"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="leLabel">
       <property name="text">
        <string>Lewis Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LeLE"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>OKButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>effButton</tabstop>
  <tabstop>effLine</tabstop>
  <tabstop>NTUButton</tabstop>
  <tabstop>estNTUButton</tabstop>
  <tabstop>NTULine</tabstop>
  <tabstop>LeLE</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
